_sass/_templates/blog-template.scss

```diff
@@ -20,7 +15,8 @@
     list-style: none;
     padding-left: 0;
     margin-left: 0;
-    margin-bottom: 5rem;
+    margin-bottom: units(5);
+    font-size: font-size($theme-body-font-family, 'sm');
```

Comment:
I think you can use a name like `'body'` here instead of referencing the variable directly